### ISSUE TITLE
Perceptual Loss - MedicalNet - doesn't work with DDP (requires_grad = True) 

### ISSUE THREAD

**Issue by virginiafdez** [Project-MONAI/GenerativeModels#272]
While attempting to use DDP with Percpetual Loss, using the option of the 3D medical net, I got an error caused by the fact that all parameters of the model object embedded in the loss had "requires_grad" = True. They shouldn't be. This causes, among other things, an error because the parameters are included in the backpropagation graph but unused. 

We should make sure that any model in the perceptual loss is marked with required_grad = False.
